ckanext/canada/schemas/presets.yaml

```diff
@@ -3166,6 +3178,10 @@ presets:
     - value: RDF
       mimetype: application/rdf+xml
       openness_score: 4
+    - value: RDF/XML
+      mimetype: application/rdf+xml 
```

Comment:
IIUC the RDF format above is the same as the RDF/XML you're adding here. XML is a more generic type